client/package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "axios": "^0.19.2",
```

Comment:
Cool library

Reply:
It's what I used in my bachelor thesis. It was recommended by a co-worker of mine who is working mostly on React or Vue projects :)